cohere/client.py

```diff
@@ -111,7 +111,8 @@ def generate(self,
                  stop_sequences: Optional[List[str]] = None,
                  return_likelihoods: Optional[str] = None,
                  truncate: Optional[str] = None,
-                 logit_bias: Dict[int, float] = {}) -> Generations:
+                 logit_bias: Dict[int, float] = {},
+                 stream: bool = False) -> Union[Generations,StreamingGenerations]:
```

Comment:
```suggestion
                 stream: bool = False,
                 ) -> Union[Generations,StreamingGenerations]:
```
nit: this way we don't have to update two lines when we append new argument

Reply:
formatter will take care of this when 4.0 is merged